apps/daimo-mobile/src/App.tsx

```diff
@@ -76,37 +78,49 @@ function AppBody() {
     (props: BottomSheetDefaultBackdropProps) => (
       <BottomSheetBackdrop
         {...props}
-        disappearsOnIndex={-1}
+        disappearsOnIndex={-0.9}
```

Comment:
...i thought the index is an integer?

Reply:
This index is traveling from 0 to -1 in a fluent way during modal movement, where it is interpolated between values given in snap points array.
So if snap points for 0 and -1 are 300 and 100 then -0.9 means that the backdrop should disappear when the offset is around 120 instead of strictly 100.